awx/ui_next/src/screens/Setting/Subscription/SubscriptionDetail/SubscriptionDetail.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t, Trans } from '@lingui/macro';
+import { Button, Label } from '@patternfly/react-core';
+import {
+  CaretLeftIcon,
+  CheckIcon,
+  ExclamationCircleIcon,
+} from '@patternfly/react-icons';
+import RoutedTabs from '../../../../components/RoutedTabs';
+import { CardBody, CardActionsRow } from '../../../../components/Card';
+import { DetailList, Detail } from '../../../../components/DetailList';
+import { useConfig } from '../../../../contexts/Config';
+import { formatDateStringUTC } from '../../../../util/dates';
+
+function SubscriptionDetail({ i18n }) {
+  const [{ license_info, version }] = useConfig();
+  const baseURL = '/settings/subscription';
+  const tabsArray = [
+    {
+      name: (
+        <>
+          <CaretLeftIcon />
+          {i18n._(t`Back to Settings`)}
+        </>
+      ),
+      link: '/settings',
+      id: 99,
+    },
+    {
+      name: i18n._(t`Subscription Details`),
+      link: `${baseURL}/details`,
+      id: 0,
+    },
+  ];
+
+  function secondsToDays(seconds) {
```

Comment:
I wonder if this would fit well in https://github.com/ansible/awx/blob/devel/awx/ui_next/src/util/dates.jsx.  Might make it easier to test out a bunch of scenarios.